packages/web-api-client/src/a11y-service-credential.ts

```diff
@@ -40,4 +33,28 @@ export class A11yServiceCredential {
 
         return request.defaults(authOptions);
     }
+
+    private async getTokenWithRetries(numRetries: number): Promise<TokenResponse> {
+        try {
+            return await this.tryGetToken();
+        } catch (err) {
+            if (numRetries > 0) {
+                return this.getTokenWithRetries(numRetries - 1);
```

Comment:
can you also add a log here that auth token call failed

Reply:
@pownkel , sorry, i have added a wrong link to fix. We should be fixing it in CredentialProvider/MSICredentialsProvider